### PR TITLE
Cleaning up various files and fixing runtime errors raised during testing

### PR DESCRIPTION
**Pull request by scriptom**
After a few tests we noticed some runtime errors in the Spring boot application. Also we are now aware we can clean up some unnecessary files. This PR is a compilation of various changes

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,12 +31,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-jdbc</artifactId>
 <!--            TODO: Includes Spring-starter-logging. Uncomment to use log4j2-->
-<!--            <exclusions>-->
```
Comment (taswartz) [inline pom.xml:34]: This would be utilizing log4j2 now? Can we change the comment above it?

File: src/main/resources/messages_en_US.properties [hunk 2]
```diff
@@ -1,58 +1 @@
...
-
-gui.combo.initialValue=NOT_SELECTED
-gui.combo.initialDisplay=--- Select ---
-
-validation.field.isrequired={0} is required
-validation.field.numeric=Please enter valid numeric value for {0}
-
-#test property for unit test making sure this .properties file is being read to messageSource
-core.test.property=core property
+# Add any validation messages you want in English
```
Comment (taswartz) [inline src/main/resources/messages_en_US.properties:1]: Did this not fail unit tests???
I thought we used `core.test.property` in one of them. But maybe that was the SDK

Comment (taswartz) [reply src/main/resources/messages_en_US.properties:1]: Ah. I see you moved it to test resources. I like that a lot more

File: src/main/java/com/client/core/base/util/Utility.java [hunk 3]
```diff
@@ -20,23 +21,13 @@
...
 import java.text.SimpleDateFormat;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Date;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TimeZone;
+import java.util.*;
```
Comment (taswartz) [inline src/main/java/com/client/core/base/util/Utility.java:30]: Usually standard to try to not import all of java util packages and only the ones we need

**Comment by taswartz** [comment]
This all looks good to me! Let me know if you decide to fix anything else up, but I think we are ready to ship this!

File: pom.xml
Comment (taswartz) [inline pom.xml:1]: Can we also remove the other extraneous comments I left in the pom.xml file? (i.e line 69 and the other TODO)

File: src/main/java/com/client/core/base/util/Utility.java (hunk 3 above)
Comment (scriptom) [reply src/main/java/com/client/core/base/util/Utility.java:30]: This is my IDE automatically doing this. I'll disable the option.

**Pull request merged**